src/scripts/extensions/clipperInject.ts

```diff
@@ -228,7 +226,7 @@ export class ClipperInject extends FrameInjectBase<ClipperInjectOptions> {
 				this.uiCommunicator.callRemoteFunction(Constants.FunctionKeys.escHandler);
 			}
 			if (oldOnKeyDown) {
-				oldOnKeyDown(event);
+				oldOnKeyDown.call(document, event);
```

Comment:
I haven't found a ton of references to this yet, though I did find one that described doing it this way.  We'll run it through the gambit of browsers and make sure everything is gravy.  Thanks!